client/my-sites/guided-transfer/issues-notices.jsx

```diff
@@ -26,10 +26,7 @@ class IssuesNotices extends Component {
 				{ premiumThemeIssue && ! premiumThemeIssue.prevents_transfer && (
 					<Notice status="is-warning" showDismiss={ false }>
 						{ translate(
-							`Your site uses a Premium Theme that can't be
-						transferred. Continuing will automatically activate the
-						default theme, or you can
-						{{a}}choose a free theme{{/a}}.`,
+							"Your site uses a Premium Theme that can't be transferred. Continuing will automatically activate the default theme, or you can {{a}}choose a free theme{{/a}}.",
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **5** times:
`translate( 'Your site uses a Premium Theme that can't be
						transferred. Continuing will automatically activate the
						default theme, or you can
						{{a}}choose a free theme{{/a}}.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/remove-unnecessary-whitespaces-from-translations)